webpack.config.js

```diff
@@ -211,9 +211,6 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 			],
 		},
 		module: {
-			// avoids this warning:
-			// https://github.com/localForage/localForage/issues/577
-			noParse: /[\/\\]node_modules[\/\\]localforage[\/\\]dist[\/\\]localforage\.js$/,
```

Comment:
The webpack warning that originally caused this issue doesn't seem to be in current versions any more.